Firestore/core/src/model/field_value_options.h

```diff
@@ -31,23 +31,16 @@ class FieldValueOptions {
    * Creates an FieldValueOptions instance that specifies deserialization
    * behavior for pending server timestamps.
    */
-  FieldValueOptions(ServerTimestampBehavior server_timestamp_behavior,
-                    bool timestamps_in_snapshots_enabled)
-      : server_timestamp_behavior_(server_timestamp_behavior),
-        timestamps_in_snapshots_enabled_(timestamps_in_snapshots_enabled) {
+  explicit FieldValueOptions(ServerTimestampBehavior server_timestamp_behavior)
```

Comment:
What do you think about removing this class? I can see arguments going either way, so I have no actual opinion here.

Reply:
I'm disinclined toward removing the class. Admittedly it doesn't serve much purpose, but we've repeatedly mooted additional snapshot options and those would potentially feed into here.